wa/workloads/gfxbench/__init__.py

```diff
@@ -0,0 +1,61 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import re
+
+from wa import ApkUiautoWorkload, WorkloadError, Parameter
+
+
+class Gfxbench(ApkUiautoWorkload):
+
+    name = 'gfxbench-corporate'
+    package_names = ['net.kishonti.gfxbench.gl.v50000.corporate']
+    regex_matches = [re.compile(r'Car Chase score (.+)'),
+                     re.compile(r'Car Chase Offscreen score (.+)'),
+                     re.compile(r'Manhattan 3.1 score (.+)'),
+                     re.compile(r'1080p Manhattan 3.1 Offscreen score (.+)'),
+                     re.compile(r'1440p Manhattan 3.1 Offscreen score (.+)'),
+                     re.compile(r'Tessellation score (.+)'),
+                     re.compile(r'Tessellation Offscreen score (.+)')]
+    description = '''
+    Execute a subset of graphical performance benchmarks
+
+    Test description:
+    1. Open the gfxbench application
+    2. Execute Car Chase, Manhattan and Tessellation benchmarks
+
+    '''
+
+    def __init__(self)
+        self.gui.timeout = self.timeout
```

Comment:
Also should add in the user parameter to allow configuration of the time-out.
e.g. 
```
parameters = [
  Parameter('timeout', kind=int, default=X, description=X),
]